src/enum/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './enum-file-format'
```

Comment:
@apolkingg8 This is the main refactoring, which I'm not 100% sure about. My goal is to modularize the features a little bit, though the code isn't exactly organized into "features" at the moment. Basically, if we only import from modules instead of files, we are free to easily refactor internal module implementations.

Ideally I would like to use tsconfig paths, e.g. `import ... '@config'` instead of `import ... '../config`, but I couldn't get it to work with the web build system when I tried briefly. It would also be nice to have eslint enforce a rule about importing from modules.